go/def.bzl

```diff
@@ -529,6 +529,13 @@ go_library_attrs = go_env_attrs + {
     ),
 }
 
+_crosstool_attrs = {
+    "_crosstool": attr.label(
+        default = Label("//tools/defaults:crosstool"),
+        cfg = HOST_CFG,
```

Comment:
Technically you should ask for the target configuration but that's does not really matter now.